app/assets/locales/en.json

```diff
@@ -25,6 +25,7 @@
   "created_at": "Created at",
   "no_result_search_input": "Could not find any results for \"{{ searchInput }}\"",
   "action_permanent": "This action cannot be undone.",
+  "invalid_url": "Invalid URL",
```

Comment:
Can we try placing this somewhere a bit more inline with how the others are placed?